### ISSUE TITLE
Autocomplete bracket inside word (function call)

### ISSUE THREAD

**Issue by SonGokussj4** [rubberduck-vba/Rubberduck#5004]
**Description**
Initialize function call with an open bracket "in the word"

**To Reproduce**
1. `Debug.Print "dict: " & dictToStr|this.dict`
2. Press `(`

**Expected behavior**
`Debug.Print "dict: " & dictToStr(|this.dict`
or
`Debug.Print "dict: " & dictToStr(this.dict)|`

**Resulting behavior**
`Debug.Print "dict: " & dictToStr((|); this.dict`

**Comment by retailcoder** [comment]
I don't think this can work as specified, off a *positional* selection; it amounts to SCP mind-reading the intent of the user here, ...with a risk of annoyingly getting it wrong.

Now, considering `[` marks where the selection starts, and `]` marks where the selection ends, then SCP could very well turn this:

    Debug.Print "dict: " & dictToStr[this.dict]

Into this (given a `(` keypress):

    Debug.Print "dict: " & dictToStr(this.dict)|
